src/Orckestra.Composer.Website/StartupHandler.cs

```diff
@@ -186,7 +186,6 @@ private static void RegisterFunctions(FunctionCollection functions)
             functions.RegisterAction<MembershipController>("ChangePasswordBlade", "Composer.Membership.ChangePassword");
 
             functions.RegisterAction<MyAccountController>("AccountHeader", "Composer.MyAccount.AccountHeader");
-            functions.RegisterAction<MyAccountController>("UpdateAccount", "Composer.MyAccount.UpdateAccount");
```

Comment:
If you remove this, you also need to clean UpdateAccount in MyAccountController, as it is not used now